packages/form-integrations/test/form-integrations.test.js

```diff
@@ -1,7 +1,7 @@
 import { expect, fixture, html } from '@open-wc/testing';
 import './helpers/umbrella-form.js';
 
-// Test umbrella form
+// Test umbrella form.
```

Comment:
Is this a linter or a newly proposed standard ? :) (the dot)

Reply:
Making sure this package gets released (.md gets ignored by lerna but we actually want it to be released so I can upgrade to latest lion in our extension and extend the .md)